source/Energinet.Charges.Libraries/source/Energinet.DataHub.Charges.Clients.Tests/DefaultChargeLink/Mappers/CreateDefaultChargeLinksInboundMapperTests.cs

```diff
@@ -23,34 +23,32 @@
 namespace Energinet.DataHub.Charges.Clients.CreateDefaultChargeLink.Tests.DefaultChargeLink.Mappers
 {
     [UnitTest]
-    public class CreateDefaultChargeLinksFailedOutboundMapperTests
+    public class CreateDefaultChargeLinksInboundMapperTests
     {
         private readonly Fixture _fixture;
 
-        public CreateDefaultChargeLinksFailedOutboundMapperTests()
+        public CreateDefaultChargeLinksInboundMapperTests()
         {
             _fixture = new Fixture();
         }
 
         [Fact]
-        public void Convert_WhenCalled_ShouldMapToProtobufWithCorrectValues()
+        public void Convert_WhenCalled_ShouldMapToDtoWithCorrectValues()
```

Comment:
Yes, it should but does it?
```suggestion
        public void Convert_WhenCalled_MapsToDtoWithCorrectValues()
```